LocalReceiptParsing

```diff
@@ -0,0 +1 @@
+Sources/LocalReceiptParsing
```

Comment:
Symlink hack FTW.

Reply:
Do docC docs continue to render well after this change? Asking because I ran into some issues with folder structure before and that's why I ended up having a `Sources` folder in the first place (although it also happened to be a good thing)

Reply:
Docs are WIP, but yeah:
![Screenshot 2022-12-14 at 11 10 04](https://user-images.githubusercontent.com/685609/207691224-b9ec3207-151a-42fd-b2ff-020845bf656c.png)

At least these docs are generated correctly for the `Purchases` package. Not sure about the `ReceiptParser` package but if this doesn't work I'd be out of ideas :/